front-end/src/app/shared/models/transaction.model.ts

```diff
@@ -0,0 +1,46 @@
+import { BaseModel } from './base.model';
+import { Contact } from './contact.model';
+import { MemoText } from './memo-text.model';
+
+export abstract class Transaction extends BaseModel {
+  id: string | undefined;
+
+  // FECFile spec properties
+
+  form_type: string | undefined;
+  filer_committee_id_number: string | undefined;
+  transaction_id: string | null = null; // This is a required field and must exist
```

Comment:
i changed this to `string | undefined` for a couple reasons
- transaction_id is not a required field in the payload because of new transactions
- there was nothing gained by having this be null while the other fields were undefined
- by sending undefined, i was able to improve the transaction_id generation on the backend.

Reply:
This change has been made.